util.ts

```
@@ -39,6 +39,9 @@ Deno.test("convertToOllamaEndpoint", async () => {
   assertEquals(result, "http://localhost:11434/v1/chat/completions");
 });
 
+export const isCursorRequest = (origin: string) => origin?.toLowerCase() === `vscode-file://vscode-app`;
+
+export const isOpenAIModel = (model: string) => match(_`gpt-${_("v")}`, model) != null;
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)